test/e2e/landing-page.spec.ts

```diff
@@ -1,11 +1,13 @@
 import { test, expect } from '@playwright/test';
+import { deleteAllCertificates } from '../../app/models/certificate.server';
 import { loggedInAsUser } from './utils';
 
 test.describe('Landing Page', () => {
   loggedInAsUser();
 
   test.beforeEach(async ({ page }) => {
-    await page.goto('/');
+    await deleteAllCertificates();
+    return await page.goto('/');
```

Comment:
Sorry, why is this here? Also, I don't think you need the return here

Reply:
You don't need the `await`